File: kickStart (Google)/Practice-1 2022/starter_code/starter_code.py
def game_status(number_of_days, vistors):
  # TODO: implement this method to determine the status of the game board
  return ""

def main():
  test_cases = int(raw_input())
  for test_case in range(1, test_cases + 1, 1):
    board_size = int(raw_input())
    board = []
    for _ in range(board_size):
      board.append(list(raw_input().strip()))

    ans = game_status(board_size, board)

    print("Case #{}: {}".format(test_case, ans))

if __name__ == "__main__":
  main()

File: kickStart (Google)/Practice-1 2022/starter_code/starter_code.cc
#include <bits/stdc++.h>
using namespace std;

string FindBoardStatus(int size, char* board) {
  // TODO: Complete this function to find the status of the board.

  return "";
}

int main() {
  int t;
  cin >> t;
  for (int tc = 1; tc <= t; ++tc) {
    int n;
    cin >> n;
    char board[100][100];
    for (int r = 0; r < n; ++r) {
      for (int c = 0; c < n; ++c) {
        cin >> board[r][c];
      }
    }
    cout << "Case #" << tc << ": " << FindBoardStatus(n, board) << endl;
  }
  return 0;
}
